### DISCUSSION TITLE
JSON dot notation value type

### DISCUSSION THREAD

**Discussion by ActxLeToucan** [FreshRSS/FreshRSS#6626] [Q&A]
Hi,
I was wondering if other value types than string are supported with the JSON dot notation.

I created a feed from a JSON with dot notation. It's a bit too basic for a perfect output, but it works and it's faster than writing code just to convert a JSON to a feed.
However, it seems like if the value of the dot-notated element is not a string, it doesn't work.
I've just realized that the date was wrong in my feed (fetch datetime instead of the parsed value). In the JSON, the value is a Unix timestamp represented by an integer. So I tried to put that element in the content field just to be sure: it's empty, there is no article content. Same thing for boolean values.
Am I doing something wrong, or is it just not possible?

**Comment by Alkarex** [comment]
Hello, 
Could you provide an example, to reproduce the problem your are facing?

**Reply by ActxLeToucan** [reply]
URL: https://www.webtoons.com/en/community/post/list?communityAuthorId=o25wi&postAfter=&pageSize=25
* articles: `posts`
* title: (empty)
* content: `body`
* link: `linkUrl`
* image: (empty)
* author: `createdBy.name`
* date: `createdAt` <- it's a number in JSON
* format: `U`
* tags: (empty)
* id: `id`

If you put `createdAt`, or other number (`emotionTotalCount`, `imageTotalCount`, ...) in content field, you'll see there is no content.

**Comment by Alkarex** [comment]
Should be fixed in https://github.com/FreshRSS/FreshRSS/pull/6631
Tests welcome
